azure-pipelines-official.yml

```diff
@@ -299,7 +299,7 @@ stages:
 
 - stage: insert
   dependsOn:
-  - build
+  - publish_using_darc
```

Comment:
found this name at https://github.com/dotnet/roslyn/blob/79d9522434143d473829192e78f31e02ddc360a3/eng/common/templates/post-build/post-build.yml#L244